components/errors/ErrorNotificationDisplay.tsx

```diff
@@ -35,6 +38,7 @@ export function ErrorNotificationDisplay ({ errors, issueDataState }: ErrorNotif
         pl={{ base:"30px", sm:"30px", md:"60px", lg:"120px" }}
         pt="4rem"
         pb="1rem"
+        padding="0.25rem"
```

Comment:
This will overwrite the padding-top and padding-bottom set by chakra-ui 'pt' & 'pb'